src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -955,6 +969,7 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
         <TradeAssetSelect
           assetId={thorchainAssetId}
           onAssetChange={handleAssetChange}
+          isReadOnly
```

Comment:
Not strictly related but this was spotted by @MBMaria - sneaking this in this PR